src/oop/programs_with_multiple_classes/medium/q7/Cupboard.java

```diff
@@ -0,0 +1,14 @@
+package oop.programs_with_multiple_classes.medium.q7;
+
+public class Cupboard {
```

Comment:
Can we change this question to not be cupboard/ kitchen themed? The previous and next questions are too similar to this one